portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,63 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+
+
+
+window.onload = function(){
+    var popupBackground = document.getElementById("popup-background")
```

Comment:
Default to use `const`, until you actually need to mutate the variable, then change to `var`.

Reply:
Same for all `var`s below.